crypto/CMakeLists.txt

```diff
@@ -490,14 +490,14 @@ endif()
 SET_TARGET_PROPERTIES(crypto PROPERTIES LINKER_LANGUAGE C)
 
 if(WIN32)
-  target_link_libraries(crypto ws2_32)
+  target_link_libraries(crypto PUBLIC ws2_32)
 endif()
 if(NOT WIN32 AND NOT ANDROID)
-  target_link_libraries(crypto pthread)
+  target_link_libraries(crypto PUBLIC pthread)
 endif()
 
 if(FIPS)
-  target_link_libraries(crypto jitterentropy)
+    target_link_libraries(crypto PRIVATE jitterentropy)
```

Comment:
* Nit: extra space.
* Why `PRIVATE` has to be used here? Can the undefined(empty) be used?
https://stackoverflow.com/questions/51396608/what-is-default-target-link-libraries-privacy-setting

```
# aws-lc still supports old version CMake.
cmake_minimum_required(VERSION 2.8.12)
```

Reply:
I'll remove the space there (my settings on VIM apparently modified that automatically). We need private because prior to specifying that this was a private dependency it was behaving as if it was public (complaining that there was no export of these build artifacts). In order to make this PRIVATE we needed to specify all the settings (since you need to either explicitly set the strategy on all of the libraries for a specific target or none of them).